### PR TITLE
Bible Tweak

### PR DESCRIPTION
**Pull request by Light-Weave**
## Description
The bible's healing has been reduced from 10 brute+10 burn to 2 brute+2 burn, and takes time to use.

## Motivation and Context
People were taking Preacher and using the role as a combat medic, using the book as a source of infinite healing rather than a roleplay tool. If you're a Preacher, start Preachin', don't start bonkin' folks with a book to close up their gunshot wounds mid-fight. (It can still heal for a total of 28 HP under optimal conditions, though.)

## How Has This Been Tested?
Ayup.

## Changelog (neccesary)
:cl:
balance: The bible now takes time to heal people, and heals less per use.
/:cl:

### CODE REVIEW

**Comment by Light-Weave** [comment]
travis is broken

**Comment by improvedname** [comment]
>removing the self smacking sizzling.
Healing yourself above others is an sinful thought and should be punished by god himself by burning your book in your hands

**Comment by Light-Weave** [comment]
Oh, previously it burned you if you tried to use it on *anyone* as non-preacher. Now you just club them over the head with a book.

**Comment by improvedname** [comment]
This is alot of stealthchanges.
You are making the book usable by anyone.
Don't forget this allows anyone to cause braindamage by going around booksmacking people which is lowrp.

My solution would be is putting a 10 second progress bar on it. Something like you sit down praying for x. Bar runs up and it heals you. Or nothing happens

**Comment by improvedname** [comment]
And don't forget you are making this gauranteed healing too. No braindamage chance at all. This only justifies the preacher go biblesmacking people. Increasing the litteral reason why he should be an combat medic

**Comment by Light-Weave** [comment]
Check the code again. No brain damage, and only preachers can heal people with it (and can't heal themselves), with a 10s progress bar.

**Comment by GhostEcho** [comment]
Not stealth changes, been discussing in coder channels

**Comment by Light-Weave** [comment]
TL;DR of how the bible works now:
If you're a preacher, you can hit someone with it, and after a short delay, heal them for a couple of points of damage to every limb. There's no brain damage involved, but the healing is minimal and takes time to use- it's more for roleplay purposes.

TL;DR of how the bible worked before:
Run up to someone with a helmet, spamclick them 20 times, they're on full HP.

File: code/game/objects/items/storage/book.dm [hunk 1]
```diff
@@ -115,30 +125,19 @@ GLOBAL_LIST_INIT(bibleitemstates, list("bible", "koran", "scrapbook", "bible",
 	if(user.mind && (user.mind.isholy))
 		chaplain = 1
 
-	if(!chaplain)
```
Comment (LetterN) [inline code/game/objects/items/storage/book.dm]: leave it to chaplains, not everyone in the wasteland is holy

Comment (Light-Weave) [reply code/game/objects/items/storage/book.dm]: still needs a chaplain to use, but it doesn't burn you alive if you're *not* a chaplain now
